context-linux/src/usr/bin/cloudinit_cc_run_cmd.rb

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env ruby
+
+# -------------------------------------------------------------------------- #
+# Copyright 2002-2024, OpenNebula Project, OpenNebula Systems                #
+#                                                                            #
+# Licensed under the Apache License, Version 2.0 (the "License"); you may    #
+# not use this file except in compliance with the License. You may obtain    #
+# a copy of the License at                                                   #
+#                                                                            #
+# http://www.apache.org/licenses/LICENSE-2.0                                 #
+#                                                                            #
+# Unless required by applicable law or agreed to in writing, software        #
+# distributed under the License is distributed on an "AS IS" BASIS,          #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   #
+# See the License for the specific language governing permissions and        #
+# limitations under the License.                                             #
+#--------------------------------------------------------------------------- #
+
+require 'fileutils'
+
+module CloudInit
+
+    ##
+    # RunCmd class implements the runcmd cloud-config directive.
+    ##
+    class RunCmd
+
+        TMP_DIR_PATH = '/var/lib/one-context/tmp/runcmd'
```

Comment:
TODO: This path is being created in context.d/net-97-start-script, should be changed by other one